src/components/Recaptcha.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div>
+    <vue-recaptcha :sitekey="this.publicKey"
```

Comment:
You shouldn't need to specify the `this` context here. By default Vue uses `this` as the context, so you set this prop via: `:sitekey="publicKey"`.

Reply:
I have made the change.